wurst/_handles/Effect.wurst

```diff
@@ -206,6 +206,18 @@ public function effect.clearSubAnimations()
 public function effect.removeSubAnimation(subanimtype whichSubAnim)
 	BlzSpecialEffectRemoveSubAnimation(this, whichSubAnim)
 
+/** Get effect scale.*/
+public function effect.getScale() returns real
+	return BlzGetSpecialEffectScale(this)
+
+/** Set effect matrix scale*/
```

Comment:
why is it matrix but takes vector 😕

Reply:
Is there any three-int tuple? if not, I can create one then...

Reply:
maybe call it public function effect.setScale(vec3 scaleVector)
and forget about blizzard misnaming things
maybe it will be changed after ptr

Reply:
Oh, just noticed, nvm my old comment, wil lchange the name a bit